google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/FindingSnippets.java

```diff
@@ -353,12 +370,18 @@ static TestIamPermissionsResponse testIamPermissions(SourceName sourceName) {
    * Group all findings under an organization across all sources by their specified properties (e.g.
    * category).
    *
-   * @param organizationName The organizatoin to group all findings for.
+   * @param organizationName The organization to group all findings for.
```

Comment:
Should there be a colon after organizationName?: "organizationName: The organization to..."